app/views/home/_schedule.html.haml

```diff
@@ -1,9 +1,10 @@
-- event_schedule = Schedule.new("1Ssak3FU6V0sJP3uPZCCsk8RITu6wW6lvaA6wlSGSOuI", 0)
```

Comment:
I think this is supposed to remain `1Ssak3FU6V0sJP3uPZCCsk8RITu6wW6lvaA6wlSGSOuI` - otherwise when I load the page, it loads last year's schedule